spring-cloud-gcp-samples/spring-cloud-gcp-trace-sample/pom.xml

```diff
@@ -44,6 +44,12 @@
             <artifactId>spring-cloud-gcp-starter-logging</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.zipkin.reporter2</groupId>
+            <artifactId>zipkin-reporter-brave</artifactId>
+            <version>2.15.2</version>
```

Comment:
Odd that it's needed in a sample -- I'd expect our trace starter to bring this dependency in.

Reply:
Good point, adding to the starter instead of the sample.